File: CS/ExtendedDrillThroughReport/IDrillThroughReportHelper.cs
using DevExpress.XtraPrinting.Preview;
using DevExpress.XtraReports.UI;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ExtendedDrillThroughReport {
    public interface IDrillThroughReportHelper {
        DocumentViewer Viewer { get; set; }
        XtraReport ParentReport { get; set; }
    }
}

File: VB/ExtendedDrillThroughReport/IDrillThroughReportHelper.vb
Imports DevExpress.XtraPrinting.Preview
Imports DevExpress.XtraReports.UI
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace ExtendedDrillThroughReport
	Public Interface IDrillThroughReportHelper
		Property Viewer() As DocumentViewer
		Property ParentReport() As XtraReport
	End Interface
End Namespace
